metricbeat/scripts/config_collector.py

```diff
@@ -61,6 +70,20 @@ def collect(short=False):
     # output string so it can be concatenated
     print config_yml
 
+# Makes sure every title line is 80 chars long
+def get_title_line(title):
+    dashes = (80 - 10 - len(title)) / 2
+
+    line = "#"
+    for i in range(0, dashes):
+        line += "-"
```

Comment:
You missed the once in lifetime opportunity to use the the multiply operator for strings: `"-" * dashes` :-). Just kidding.